From 5ef2f499e8539425e236122b78d9a4c977411d8a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Fri, 22 Mar 2024 02:16:09 +0100
Subject: [PATCH] Remove gnome-session options that are no longer available

Since gnome-session 46, systemd became a hard dependency, '--systemd' and '--builtin' options are no longer available.

https://gitlab.gnome.org/GNOME/gnome-flashback/-/issues/94
---
 data/Makefile.am | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/data/Makefile.am b/data/Makefile.am
index 08ee025..9cada99 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -20,18 +20,12 @@ libexec_SCRIPTS = \
 	gnome-flashback-metacity \
 	$(NULL)
 
-if ENABLE_SYSTEMD_SESSION
-session_management=--systemd
-else
-session_management=--builtin
-endif
-
 gnome-flashback-metacity: Makefile
 	$(AM_V_GEN) (echo '#!/bin/sh' && echo '' && \
 	             echo 'if [ -z $$XDG_CURRENT_DESKTOP ]; then' && \
 	             echo '  export XDG_CURRENT_DESKTOP="GNOME-Flashback:GNOME"' && \
 	             echo 'fi' && echo '' && \
-	             echo -n 'exec gnome-session $(session_management) ' && \
+	             echo -n 'exec gnome-session ' && \
 	             echo -n '--session=gnome-flashback-metacity ' && \
 	             echo '--disable-acceleration-check "$$@"') > $@
 	$(AM_V_at) chmod a+x $@
@@ -50,7 +44,7 @@ gnome-flashback-compiz: Makefile
 	             echo 'if [ -z $$XDG_CURRENT_DESKTOP ]; then' && \
 	             echo '  export XDG_CURRENT_DESKTOP="GNOME-Flashback:GNOME"' && \
 	             echo 'fi' && echo '' && \
-	             echo -n 'exec gnome-session $(session_management) ' && \
+	             echo -n 'exec gnome-session ' && \
 	             echo '--session=gnome-flashback-compiz "$$@"') > $@
 	$(AM_V_at) chmod a+x $@
 
-- 
2.44.0

